web/client/components/manager/rulesmanager/rulesgrid/filterRenderers/UsersFilter.jsx

```diff
@@ -16,7 +16,8 @@ import { createSelector } from 'reselect';
 import { filterSelector } from '../../../../../selectors/rulesmanager';
 import { error } from '../../../../../actions/notifications';
 const selector = createSelector(filterSelector, (filter) => ({
-    selected: filter.username
+    selected: filter.username,
+    anyFieldVal: filter.anyUser
```

Comment:
following other approaches this should be called userAny

```suggestion
    anyFieldVal: filter.userAny
```